packages/devtools_app/lib/src/framework/release_notes/README.md

```diff
@@ -1,9 +1,9 @@
-## Writing DevTools release notes
+## Generating Release notes
 - Release notes for DevTools are hosted on the flutter website (see [archive](https://docs.flutter.dev/development/tools/devtools/release-notes)).
-- To add release notes for the latest release, create a PR with the appropriate changes for your release
-    - The [release notes template](release-notes-template.md) can be used as a starting point
-    - see example [PR](https://github.com/flutter/website/pull/6791).
+- To add release notes for the latest release, create a PR with the appropriate changes for your release.
```

Comment:
this should point us to creating an issue on the flutter/website repo - not a PR. Our tech writers will take the content we give them, polish it, and publish the release notes on the website.